.github/workflows/guestconfiguration-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Guest Configuration Test
+
+on: [workflow_dispatch, pull_request]
```

Comment:
If running this on PRs we should be able to add this as a separate independent job in `ci.yml` and reuse the CI report workflow for generating the test report(s). This will help with the "parent" workflow run that GitHub chooses for the check-run that is created by the report action.

Reply:
Good idea